cache/s3proxy/s3proxy.go

```diff
@@ -68,13 +69,23 @@ func New(
 		log.Fatalf("Failed to determine s3proxy credentials")
 	}
 
+	secure := !DisableSSL
+	tr, err := minio.DefaultTransport(secure)
+	if err != nil {
+		log.Fatalf("Failed to create default transport: %v", err)
```

Comment:
I think this would make it easier to identify where the error is in the code:
```suggestion
		log.Fatalf("Failed to create default minio transport: %v", err)
```